controllers/auth_config_controller.go

```diff
@@ -234,10 +234,22 @@ func (r *AuthConfigReconciler) translateAuthConfig(ctx context.Context, authConf
 				})
 			}
 
+			headers := make([]common.JSONProperty, 0, len(genericHttp.Headers))
+			for _, header := range genericHttp.Headers {
+				headers = append(headers, common.JSONProperty{
```

Comment:
Having `JsonProperty` and `JSONProperty` is a little bit confusing. Small thing.. just for the record.